presto-client/src/main/java/io/prestosql/client/SpnegoHandler.java

```diff
@@ -100,7 +102,16 @@ public SpnegoHandler(
         this.keytab = requireNonNull(keytab, "keytab is null");
         this.credentialCache = requireNonNull(credentialCache, "credentialCache is null");
 
-        kerberosConfig.ifPresent(file -> System.setProperty("java.security.krb5.conf", file.getAbsolutePath()));
+        kerberosConfig.ifPresent(file -> {
+            String newValue = file.getAbsolutePath();
+            String currentValue = System.getProperty("java.security.krb5.conf");
+            checkState(
+                    currentValue == null || Objects.equals(currentValue, newValue),
```

Comment:
This can be
```java
newValue.equals(currentValue)
```
Since `getAbsolutePath()` cannot return null

Reply:
I know. I still prefer `Objects.equals` because it "parenthises" the expression better and to me. Also, to me, `.equals()` triggers "can this be null?" deoptimization when reading